docs/reference/contributing/connectivity/LoRaPortingGuide.md

```diff
@@ -1,71 +1,69 @@
-### Porting LoRaWAN stack & LoRa RF drivers
+<h2 id="lora-port">LoRa</h2>
```

Comment:
This could be `LoRaWAN porting guidelines`. LoRa is the name of a radio technology.